pkg/api/v1/router_bom.go

```diff
@@ -0,0 +1,138 @@
+package serverservice
+
+import (
+	"github.com/pkg/errors"
+	"github.com/tealeg/xlsx"
+	"golang.org/x/exp/maps"
+)
+
+const (
+	// column name
+	serialNumName string = "SERIALNUM"
+	subItemName   string = "SUB-ITEM"
+	subSerialName string = "SUB-SERIAL" // value of the sub-item
+	// interested catogories in sub-item
+	aocMacAddrName string = "MAC-AOC-ADDRESS"
+	bmcMacAddrName string = "MAC-ADDRESS"
+	numDefipmiName string = "NUM-DEFIPMI"
+	numDefipwdName string = "NUM-DEFPWD"
```

Comment:
I hate bringing up variable names, but I feel like these names don't add much here. Consider `ipmiFieldName` (and likewise `pwdFieldName`)?